data/prefecture.yml

```diff
@@ -1,5 +1,6 @@
-# JIS X 0402 で定義されている都道府県コードと都道府県名をベースに
+# JIS X 0401 で定義されている都道府県コードと都道府県名をベースに
```

Comment:
There are other comments with '0402'. Could you replace them with '0401'? 🙏 
https://github.com/search?q=repo%3Achocoby%2Fjp_prefecture%200402&type=code

Reply:
Thank you for catching this🙏 I fixed other comments.

Reply:
memo) There are 402s remaining below.
<img width="345" alt="remain" src="https://github.com/user-attachments/assets/5c68f50b-c8ea-4713-aeab-79ca3f264a6a" />

Reply:
Thank you. changed 🙆